File: challenges/parent.py
class Parent:
     def __init__(self, name):
             self.name = name
     def spam(self):
             print("Parent.spam", self.name)
     def grok(self):
             print("Parent.grok")

class Child(Parent):
	def yow(self):
		print("Child.yow()")

File: challenges/parent.js
class Parent {
	constructor(name) {
		this.name = name;
	}
	
	spam() {
		console.log("Parent.spam %s", this.name);
	}
	
	grok() {
		console.log("Parent.grok");
		this.spam();
	}
}

class Child extends Parent {
	constructor(name) {
		super(name);
	}

	yow() {
		console.log("Child.yow");
	}	
}

module.exports = {
	Parent: Parent,
	Child: Child
}
